fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergConnector.java

```diff
@@ -0,0 +1,68 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.connector.iceberg;
+
+import com.google.common.base.Preconditions;
+import com.starrocks.common.Config;
+import com.starrocks.common.DdlException;
+import com.starrocks.common.util.Util;
+import com.starrocks.connector.Connector;
+import com.starrocks.connector.ConnectorContext;
+import com.starrocks.connector.ConnectorMetadata;
+import com.starrocks.server.GlobalStateMgr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Map;
+
+public class IcebergConnector implements Connector {
+    private static final Logger LOG = LogManager.getLogger(IcebergConnector.class);
+
+    private static final String ICEBERG_METASTORE_URIS = "iceberg.catalog.hive.metastore.uris";
+    private static final String ICEBERG_IMPL = "iceberg.catalog-impl";
+    private final Map<String, String> properties;
+    private final String catalogName;
+    private String resourceName;
+    private ConnectorMetadata metadata;
+
+    public IcebergConnector(ConnectorContext context) {
+        this.catalogName = context.getCatalogName();
+        this.properties = context.getProperties();
+        validate();
+        onCreate();
+    }
+
+    public void validate() {
+        this.resourceName = Preconditions.checkNotNull(properties.get(ICEBERG_METASTORE_URIS),
+                "%s must be set in properties when creating iceberg catalog", ICEBERG_METASTORE_URIS);
+        Util.validateMetastoreUris(resourceName);
```

Comment:
we needn't resourceName in IcebergConnector, because we don't use hive meta cache when getting schemas or statistics.